src/components/MedCentersPage/MedCentersPage.tsx

```diff
@@ -9,18 +9,23 @@ import MedCentersMap from './MedCentersMap/MedCentersMap';
 const MedCentersPage: React.FC = () => {
   const dispatch = useDispatch();
 
+  const [hoverMedCenter, setHoverMedCenter] = useState<string>('');
```

Comment:
what for is used?

Reply:
when the user hovers the mouse over the card, the medical center changes the marker on the map

Reply:
I remember the id of the medical center and then in the map companion I compare